telos-backend/src/routes/api/journal.js

```diff
@@ -0,0 +1,46 @@
+const express = require('express');
+
+const router = express.Router();
+
+// get widgets which appear in the journal for a date range.
```

Comment:
it says date range here but the route parameter only has a single date? Is this meant to be for just a single date or are there startDate and endDate query params?

Reply:
Oops the comment is wrong, it's only meant to be a single date

Reply:
In that case looks good :)

Reply:
I think we forgot to change the comment after we changed the actual code oops. From my understanding, it makes more sense for it to be just a single date right since front-end only ever needs to retrieve one page at a time?